src/api/manifest/ManifestEditor.ts

```diff
@@ -1,6 +1,6 @@
 import { readJson, writeJson, writeJsonSync } from 'fs-extra'
 import { resolve } from 'path'
-import { CommandError } from '../error/errors'
+import { createFlowIssueError } from '../error/utils'
```

Comment:
Why in some cases this is being imported from ../error and in this case utils is being specified?

Reply:
Sorry, going to fix this